UpdatePublishedVersions.ps1

```diff
@@ -16,6 +16,8 @@ param(
     # A pattern matching all packages in the set that the versions repository should be set to.
     [Parameter(Mandatory=$true)][string]$nupkgPath)
 
+& "$PSScriptRoot\init-tools.cmd"
```

Comment:
If we're keeping init-tools.cmd in dotnet.cmd, do we also need it here?

Reply:
I wanted to remove it.  I added it here before...  I'll remove